symfony/asset-mapper/6.4/importmap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Returns the import map for this application.
+ *
+ * - "path" is a path inside the asset mapper system. Use the
+ *     "debug:asset-map" command to see the full list of paths.
+ *
+ * The "importmap:require" command can be used to add new entries to this file.
+ *
+ * This file has been auto-generated by the importmap commands.
+ */
+return [
+    'app' => [
+        'path' => 'app.js',
```

Comment:
Also, the path will now be:

./assets/app.js

The old string (the logical path) will still work. But this relative path is simpler to explain and understand and now works in 6.4.